### ISSUE TITLE
How to install the version2.0 pyshp?

### ISSUE THREAD

**Issue by Chenghui123** [GeospatialPython/pyshp#169]
How to install the version2.0 pyshp?
I replace the shapefile.py document, then it gives a error.
![image](https://user-images.githubusercontent.com/33248072/47701436-c1b49100-dc54-11e8-9219-0d105f7c8fd6.png)
So, what should i do?

**Comment by micahcochran** [comment]
You should install this package using:
`$ pip install pyshp`

I suspect this is another instance of issue #139 .  What are the field names in the shapefile "02_ZA03341003"?

**Comment by Chenghui123** [comment]
I use ```pip install pyshp``` to install, then it installed the version of 1.2.12. Then I replace the shapefile.py docunment with this https://github.com/GeospatialPython/pyshp/blob/master/shapefile.py. It is correct?

This is the field names in the shapefile "02_ZA03341003"

![image](https://user-images.githubusercontent.com/33248072/47839612-50a8e100-ddee-11e8-9c56-a817dbd4f30c.png)

**Comment by karimbahgat** [comment]
PyShp v2.0.0 is now officially released on PyPI, so a 'pip install pyshp' should work as well. 
As for the problem however, upgrading will not fix the problem with non-ascii characters as field names. As detailed in #139, the shapefile/dbf spec only allows ascii characters in field names, and this was only enforced in more recent versions including 2.0.0. 

However, I am starting to consider the idea to at least allow reading such shapefiles, and possibly also writing them combined with a warning. Especially so if other popular GIS tools/software also support/handle such files. If you would like to see this feature or have any thoughts, please comment in #139.

**Comment by sebastic** [comment]
PyShp fails to build from source with Python 2.7 due to this issue:
```
I: pybuild base:184: python2.7 setup.py clean 
running clean
removing '/home/bas/git/pkg-grass/pyshp/.pybuild/pythonX.Y_2.7/build' (and everything under it)
'build/bdist.linux-x86_64' does not exist -- can't clean it
'build/scripts-2.7' does not exist -- can't clean it
I: pybuild base:184: python3.5 setup.py clean 
Traceback (most recent call last):
  File "setup.py", line 6, in <module>
    long_description=open('README.md').read(),
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 37236: ordinal not in range(128)
E: pybuild pybuild:283: clean: plugin distutils failed with: exit code=1: python3.5 setup.py clean 
```

The following lines seems to be problematic due to not containing ASCII characters:
```
1018         >>> r.record(0) == [2, u'Ñandú']
...
1033         >>> r.record(0) == [2, u'Ñandú']
...
1043         >>> r.record(0) == [2, u'�and�']
```

`setup.py` should probably read the `README.md` differently when executed with Python 2.7.

**Comment by megies** [comment]
@sebastic I can't reproduce your problem. In any case it seems it's failing for you on Python 3.5, not Python 2.7.

Curious @sebastic, are you packaging for Debian/Ubuntu?

**Comment by sebastic** [comment]
> Curious @sebastic, are you packaging for Debian/Ubuntu?

Yes, the build was occured when updating the pyshp Debian package to 2.0.0.

The changes in #171 fix that build failure and are included as a patch in the Debian package.

**Comment by Chenghui123** [comment]
I try to install PyShp v2.0.0 with ```pip install pyshp```, then I got this error.

![image](https://user-images.githubusercontent.com/33248072/48045358-0303ee00-e1cb-11e8-8b03-9fb7079a0ac9.png)

**Comment by micahcochran** [comment]
@Chenghui123  Does the release of version 2.0.1 fix the problem?

**Comment by megies** [comment]
I believe it should

**Comment by karimbahgat** [comment]
Marking this as closed since 2.0.1 should fix it.
